internal/controller/streaming_manager.go

```diff
@@ -82,42 +84,8 @@ func ServerIsHealthy() bool {
 	return true
 }
 
-// NewStream returns a new stream.
-func NewStreams(ctx context.Context, logger *zap.SugaredLogger, conn *grpc.ClientConn) (*streamManager, error) {
-	client := pb.NewKubernetesInfoServiceClient(conn)
-
-	SendLogsStream, err := client.SendLogs(ctx)
-	if err != nil {
-		// Proper error handling here; you might want to return the error, log it, etc.
-		logger.Errorw("Failed to connect to server",
-			"error", err,
-		)
-		return &streamManager{}, err
-	}
-	SendKubernetesResourcesStream, err := client.SendKubernetesResources(ctx)
-	if err != nil {
-		// Proper error handling here; you might want to return the error, log it, etc.
-		logger.Errorw("Failed to connect to server", "error", err)
-		return &streamManager{}, err
-	}
-
-	instance := &streamClient{
-		conn:           conn,
-		resourceStream: SendKubernetesResourcesStream,
-		logStream:      SendLogsStream,
-	}
-	sm := &streamManager{
-		instance: instance,
-		logger:   logger,
-	}
-	return sm, nil
-}
-
-// BootUpStreamAndReconnect creates the clients needed to read K8s resources and
-// also creates all of the objects that help organize and pass info to the goroutines that are listing and watching
-// different resource types. This is also handling the asyc nature of listing resources and properly sending our commit
-// message on intial boot when we have the state of the cluster.
-func (sm *streamManager) BootUpStreamAndReconnect(ctx context.Context, cancel context.CancelFunc) error {
+// StreamResources handles the resource stream.
```

Comment:
```suggestion
// StreamResources handles the resource stream that streams K8 resources and any events related to them
```

Something like that?